src/child.rs

```diff
@@ -0,0 +1,140 @@
+//! Child trie host interface for Substrate runtime.
```

Comment:
Why do we need this file? Can we instead import this from the default substrate node?

Reply:
this is the file that fills the breaking change :)

Reply:
Can you explain which part fills the breaking change? Do you have a rough document that explains the actual breaking change?

Reply:
This entire document fills the breaking change. This document include the wasm host functions that are removed in rc3 since they moved towards separating child trie implementation under a `default_child_trie` namespace

Reply:
these are functions that were part of child.rs before. They refactored it to default_child_trie but our old runtime wasm still expects the old host functions. This file will plug the hole for old wasm and any newer upgrades will directly use default_child_trie instead

Reply:
Got it, thanks!